pysnooper/tracer.py

```diff
@@ -287,10 +293,10 @@ def trace(self, frame, event, arg):
 
         newish_string = ('Starting var:.. ' if event == 'call' else
                                                             'New var:....... ')
-        for name, value_repr in sorted(newish_local_reprs.items()):
+        for name, value_repr in newish_local_reprs.items():
```

Comment:
This can be taken a step further. Since we're not sorting, we don't need intermediate dicts like `newish_local_reprs`. We can just write immediately in the loop above. Kills several lines of code.

Reply:
Agreed.

Reply:
Done, https://github.com/cool-RR/PySnooper/pull/113/commits/d99921f7fd593714643b975d1c8ea254325e0ffc